src/classes/Passage.ts

```diff
@@ -146,8 +142,8 @@ export default class Passage {
    * @param {Request} req Express request
    * @return {string} UserID for Passage User
    */
-    async authenticateRequestWithCookie(req: Request): Promise<string | false> {
-        if (!req.headers.cookie) {
+    async authenticateRequestWithCookie(req: Request): Promise<string> {
+        if (!req.headers?.cookie) {
```

Comment:
why was this `?` mark added?

Reply:
This will now evaluate to undefined if req.headers doesn't exist. Before it would have caused a reference error if headers didn't exist on the request. 

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining